src/backend/main.ts

```diff
@@ -276,9 +276,9 @@ const processZoomForScreen = (zoomFactor: number) => {
   const screenSize = screen.getPrimaryDisplay().workAreaSize.width
   if (screenSize < 1200) {
     const extraDPIZoomIn = screenSize / 1200
-    return zoomFactor * extraDPIZoomIn
+    return (zoomFactor * extraDPIZoomIn - 1) / 0.2
   } else {
-    return zoomFactor
+    return (zoomFactor - 1) / 0.2
```

Comment:
instead of setting a zoom factor (like 1.2, 1.4, 0.8, etc), `setZoomLevel` expects an integer with increments of `0.2` zoom

so a zoom of 80% is 0.8 zoom factor so it's `-1` for zoom level
120% is 1.2 zoom factor so it's 1 zoom level